string name
time start_stamp
time end_stamp
